tests/unit/test_websocket_router.py

```
@@ -265,6 +265,47 @@ async def test_websocket_rejects_invalid_batch(self) -> None:
             code=status.WS_1008_POLICY_VIOLATION
         )
 
+    @pytest.mark.asyncio
+    async def test_websocket_rejects_non_owner(self, mock_ws_logger: MagicMock) -> None:
```

Ruff (0.15.12)

[warning] Unused method argument: `mock_ws_logger`

(ARG002)